assets/src/scripts/monitoring-location/store/hydrograph-data.js

```diff
@@ -0,0 +1,349 @@
+
+import {DateTime, Duration} from 'luxon';
+
+import config from 'ui/config';
+import {convertCelsiusToFahrenheit} from 'ui/utils';
+
+import {fetchGroundwaterLevels} from 'ui/web-services/groundwater-levels';
+import {fetchTimeSeries} from 'ui/web-services/instantaneous-values';
+import {fetchSiteStatistics} from 'ui/web-services/statistics-data';
+
+import {isCalculatedTemperature, getConvertedTemperatureParameter} from 'ml/iv-data-utils';
+
+const getParameterToFetch = function(parameterCode) {
```

Comment:
I don't need to make the synchronous actions external because we don't want another module to set these.